cli/app.go

```diff
@@ -2870,10 +2873,13 @@ Example:
 							Usage:     "get the logs from one of your cloud builds",
 							UsageText: createUsageText("module build logs", []string{generalFlagID}, true, false),
 							Flags: []cli.Flag{
-								&cli.StringFlag{
-									Name:     generalFlagID,
-									Usage:    "build that you want to get the logs for",
-									Required: true,
+								&AliasStringFlag{
+									cli.StringFlag{
+										Name:     moduleBuildFlagId,
```

Comment:
(flyby) the `--id` flag here was a bit confusing, and led me to pass the incorrect `module-id` as is asked for in several other module commands in the `--id` flag. I've changed it to `--build-id` to be more explicit, and added `--id` as an invisible alias to maintain backwards compatibility.